lisp/forge-core.el

```diff
@@ -174,6 +174,9 @@ argument.")
   "Visit the URL corresponding to a forge object in a browser."
   (browse-url (forge-get-url forge-object)))
 
+(cl-defgeneric forge-visit (forge-object)
```

Comment:
This cannot be used as the name of an argument because there is a class by that name and, for backward compatibility, `defclass` also defines a variable by that name whose value is the class.